packages/web-lib/hooks/useBalances.tsx

```diff
@@ -20,53 +20,75 @@ const		defaultStatus = {
 	isFetched: false,
 	isRefetching: false
 };
+const		defaultData = {
+	[toAddress('')]: {
```

Comment:
```suggestion
	[ethers.constants.AddressZero]: {
```
How about making this a bit more explicit? Calling `toAddress` with empty string feels strange, is there a reason to have that?